github/issues_labels.go

```diff
@@ -21,6 +22,13 @@ func (l Label) String() string {
 	return fmt.Sprint(*l.Name)
 }
 
+func (l Label) GetID() int {
```

Comment:
accessors should be generated using `go generate`.  /cc @gmlewis for details as this was just recently added.